src/models/link/metadata.rs

```diff
@@ -12,41 +9,125 @@ use crate::interchange::DataInterchange;
 use crate::Result;
 
 use crate::models::step::Command;
-use crate::models::{Link, Metablock, Metadata, TargetDescription, VirtualTargetPath};
+use crate::models::{
+    Convert, Metablock, PredicateVer, PredicateWrapper, StatementVer, StatementWrapper,
+    TargetDescription, VirtualTargetPath,
+};
+use crate::Error;
+use derive_getters::{Dissolve, Getters};
 
 use super::byproducts::ByProducts;
 
+#[derive(Dissolve, Getters)]
+pub struct MetaFormat {
```

Comment:
`MetaFormat` is used to indicate the output version of `Predicate` and `Statement`